src/bit-components.js

```diff
@@ -90,12 +90,40 @@ export const PreventAudioBoost = defineComponent();
 export const IgnoreSpaceBubble = defineComponent();
 export const Rigidbody = defineComponent({
   bodyId: Types.ui16,
-  collisionGroup: Types.ui32,
-  collisionMask: Types.ui32,
-  flags: Types.ui8,
-  gravity: Types.f32
+  mass: Types.f32,
+  gravity: [Types.f32, 3],
+  linearDamping: Types.f32,
+  angularDamping: Types.f32,
+  linearSleepingThreshold: Types.f32,
+  angularSleepingThreshold: Types.f32,
+  angularFactor: [Types.f32, 3],
+  type: Types.ui8,
+  activationState: Types.ui8,
+  collisionFilterGroup: Types.ui32,
+  collisionFilterMask: Types.ui32,
+  flags: Types.ui8
+});
+export const PhysicsShape = defineComponent({
+  bodyId: Types.ui16,
+  type: Types.ui8,
+  fit: Types.ui8,
+  halfExtents: [Types.f32, 3],
+  minHalfExtent: Types.f32,
+  maxHalfExtent: Types.f32,
+  sphereRadius: Types.f32,
+  cylinderAxis: Types.ui8,
+  margin: Types.f32,
+  offset: [Types.f32, 3],
+  orientation: [Types.f32, 4],
+  heightfieldData: [Types.f32],
+  heightfieldDistance: Types.f32,
+  flags: Types.ui8
 });
-export const PhysicsShape = defineComponent({ bodyId: Types.ui16, shapeId: Types.ui16, halfExtents: [Types.f32, 3] });
+export const AmmoShape = defineComponent();
```

Comment:
This is partially related to this comment: https://github.com/mozilla/hubs/pull/5978#discussion_r1128348430 
I think the way we use offset is a bit confusing. In some places like Spoke's Box collider we set the offset from spoke as the object transform: https://github.com/mozilla/Spoke/blob/9fe7af7e0b4eab5908d1ada17c06aab223c978ce/src/editor/nodes/BoxColliderNode.js#L73 that then use as the shape object in the client.
For other components like `ammo-shape` we let the user to set the offset manually and it might be interpreted as "this shape will be positioned where the attached mesh is plus whatever offset that you set here" (definitely we can improve that in the add-on side)

Part of the reason those components exist it's just to set the offsets when adding the shape as that's the only moment that we know for sure that the Object3D has been already loaded (ie. async loading of a media loader model). A possible solution would be to just set the offsets in Spoke/Blender, doing that avoid to have to do this extra step.